packages/cardano-services/src/Http/HttpServer.ts

```diff
@@ -92,9 +109,10 @@ export class HttpServer extends RunnableModule {
     for (const service of this.#dependencies.services) {
       await service.initialize();
       const serviceVersionPath = service.apiVersionPath();
-      const serviceBaseUrl = path.join(serviceVersionPath, service.slug);
-      this.app.use(serviceBaseUrl, service.router);
-      this.logger.info(`Using ${serviceBaseUrl}`);
+      const majorVersion = serviceVersionPath.split('.')[0];
+      const serviceBaseUrlRegex = new RegExp(`^${majorVersion}\\.\\d+\\.\\d+\\/${service.slug}`);
+      this.app.use(serviceBaseUrlRegex, this.checkReqVersion(serviceVersionPath), service.router);
+      this.logger.info(`Using ${serviceBaseUrlRegex}`);
```

Comment:
I am not sure what we want to display here. Maybe we should keep displaying the exact version. What do you think?

Reply:
That looks fine to me, assuming it's printing the regex in a readable way

Reply:
It looks like: `Using /^\/v15\.\d+\.\d+\/utxo/`

Reply:
That's fine by me :)